products/apache-apisix.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Apache Apisix
+category: app
+permalink: /apisix
+alternate_urls:
+-   /apache-apisix
+
+auto:
+  methods:
+  -   git: https://github.com/apache/apisix
+
+identifiers:
+  - repology: apisix
+
+releases:
+-   releaseCycle: "3.9"
+    releaseDate: 2024-03-25
+    eol: false
+    lts: false
+    latest: "3.9.1"
+    latestReleaseDate: 2024-04-24
+    link: https://github.com/apache/apisix/blob/master/CHANGELOG.md#390
+
+-   releaseCycle: "3.8"
+    releaseDate: 2024-01-15
+    eol: false
+    lts: false
+    latest: "3.8.1"
+    latestReleaseDate: 2024-04-24
+    link: https://github.com/apache/apisix/blob/master/CHANGELOG.md#380
+
+-   releaseCycle: "3.7"
+    releaseDate: 2023-11-21
+    eol: false
+    lts: false
+    latest: "3.7.0"
+    latestReleaseDate: 2023-11-21
+    link: https://github.com/apache/apisix/blob/master/CHANGELOG.md#370
+
+-   releaseCycle: "3.6"
+    releaseDate: 2023-10-04
+    eol: false
+    lts: false
+    latest: "3.6.0"
+    latestReleaseDate: 2023-10-04
+    link: https://github.com/apache/apisix/blob/master/CHANGELOG.md#360
+
+-   releaseCycle: "3.5"
+    releaseDate: 2023-09-01
+    eol: false
+    lts: false
+    latest: "3.5.0"
+    latestReleaseDate: 2023-09-01
+    link: https://github.com/apache/apisix/blob/master/CHANGELOG.md#350
+
+-   releaseCycle: "3.4"
+    releaseDate: 2023-06-30
+    eol: false
+    lts: false
+    latest: "3.4.1"
+    latestReleaseDate: 2023-07-20
+    link: https://github.com/apache/apisix/blob/master/CHANGELOG.md#340
+
+-   releaseCycle: "3.3"
+    releaseDate: 2023-05-04
+    eol: false
+    lts: false
+    latest: "3.3.0"
+    latestReleaseDate: 2023-05-04
+    link: https://github.com/apache/apisix/blob/master/CHANGELOG.md#330
+
+-   releaseCycle: "3.2"
+    releaseDate: 2023-03-06
+    eol: false
+    lts: true
+    latest: "3.2.2"
+    latestReleaseDate: 2023-07-22
+    link: https://github.com/apache/apisix/blob/master/CHANGELOG.md#320
+
+---
+
+> [Apisix](https://apisix.apache.org/) is a high-performance cloud native API gateway.
+
+Apache APISIX provides rich traffic management features like Load Balancing, Dynamic Upstream, Canary Release, Circuit Breaking, Authentication, Observability, etc.
```

Comment:
This describes the product itself. Can you change it to describe the release / support / EOL policy instead, and mention there is a notion of LTS (see https://apisix.apache.org/downloads/) ?

Reply:
Done.